imsim/stamp.py

```diff
@@ -59,7 +58,7 @@ class LSST_SiliconBuilder(StampBuilder):
                               wave_type='nm', flux_type='fphotons')
     _Nmax = 4096  # (Don't go bigger than 4096)
 
-    diffraction_psf: DiffractionPSF
+    diffraction_fft: DiffractionFFT
```

Comment:
I think this is a type hint.  Does it belong here and not above?

Reply:
Honestly, I don't understand this syntax.  It's something Geri put in.  I think it's related to DataClasses somehow, but this class isn't marked with @dataclass, so I'm not really sure what it does.

Reply:
I think this is a type hint for something like mpy to use.  It's letting the type checker know that difraction_fft is of type DiffractionFFT.  But, I have never actually used this before.

Reply:
Apparently this is a type annotation.  (Maybe you knew that already.)  https://peps.python.org/pep-0526/

Nothing breaks when I get rid of it.  And it strikes me as not very useful, tbh.  We aren't doing type annotations other places in imSim, and I suspect we mostly don't want to.  We rely on duck typing most of the time, rather than static type checking, so I think we should just remove this line.